Core/include/Acts/Seeding/SeedFinderOrthogonal.hpp

```diff
@@ -100,7 +100,7 @@ class SeedFinderOrthogonal {
    */
   template <typename input_container_t, typename output_container_t>
   void createSeeds(const input_container_t &spacePoints,
-                   std::back_insert_iterator<output_container_t> out_it) const;
+                   output_container_t &out_it) const;
```

Comment:
In order to implement the replace mechanism where we check if there is a lower quality seed that can be replaced if a certain number of seeds in reached, I needed to change the container and that's the only way I found to do it. Do you know if I can do that with the back_insert_interator?

Reply:
do you have to check them among all seeds from all space points that have been created so far? Then I think there is no better option. You may want to rename the places where you changed the back_insert_iterator to output_container_t from out_it to out_cont (since it's no longer an iterator).

Reply:
Yes. ok, I will change it to out_cont

Reply:
done